Composer/packages/client/src/pages/language-understanding/LUPage.tsx

```diff
@@ -21,17 +21,20 @@ const CodeEditor = React.lazy(() => import('./code-editor'));
 const LUPage: React.FC<RouteComponentProps<{
   dialogId?: string;
   projectId: string;
+  skillId: string;
 }>> = (props) => {
-  const { dialogId = '', projectId = '' } = props;
-  const dialogs = useRecoilValue(validateDialogsSelectorFamily(projectId));
+  const { dialogId = '', projectId = '', skillId = null } = props;
```

Comment:
is there a reason why skillId is instanitated to null and other props are'nt? Can we make the instantiation consistent?

Reply:
I originally had this as `''`, but that broke the `??` melding I was doing. If anything, I think they should all default to `null`, so I'll make that change too.

Reply:
Update on this - changing them all to `null` caused more typechecking issues downstream, so I found another solution that looks less strange than this. We need a projectId and dialogId in all cases, but the skillId is optional and gets marked as such.